app/src/components/searchWidget/budgetRangeSlider.tsx

```diff
@@ -42,8 +43,20 @@ class BudgetRangeSlider extends React.Component<SliderProps, SliderState> {
         this.props.onChange && this.props.onChange([this.props.min, sliderValue]);
     }
 
+    getStep(){
+        
+        if(this.state.currentValue>=Math.round(this.props.max*
+            (process.env.REACT_APP_SLIDER_HIGHEST_TRESHOLD?parseFloat(process.env.REACT_APP_SLIDER_HIGHEST_TRESHOLD):0.67)))
+        return process.env.REACT_APP_SLIDER_HIGHEST_STEP?parseInt(process.env.REACT_APP_SLIDER_HIGHEST_STEP):this.props.step;      
+        else if(this.state.currentValue>=Math.round(this.props.max*
+            (process.env.REACT_APP_SLIDER_MEDIUM_TRESHOLD?parseFloat(process.env.REACT_APP_SLIDER_MEDIUM_TRESHOLD):0.34)))
+        return process.env.REACT_APP_SLIDER_MEDIUM_STEP?parseInt(process.env.REACT_APP_SLIDER_MEDIUM_STEP):this.props.step;
+        else
+        return process.env.REACT_APP_SLIDER_LOWEST_STEP?parseInt(process.env.REACT_APP_SLIDER_LOWEST_STEP):this.props.step;
+    }
+
```

Comment:
Also, did you disable ESLint or never enabled 'cause still using bs code? Our rules do not allow for non-curly one line if statements